src/carousel.tsx

```diff
@@ -73,6 +73,82 @@ export const PageBullets: React.FC<PageBulletsProps> = ({currentIndex, numPages,
     );
 };
 
+const defaultGoPrev = () => {
```

Comment:
NIT: I think the name of this function should be related to the error itself (maybe something like `onMissingContextProvider`).

Then you could do:
```
const defaultGoPrev = onMissingContextProvider;
const defaultGoNext = onMissingContextProvider;
const defaultGoToPage = onMissingContextProvider;
```

Reply:
done (something similar)